docs/config.md

```diff
@@ -131,6 +131,47 @@ php_server [<matcher>] {
 }
 ```
 
+### Watching for File Changes
+
+Since workers won't restart automatically when updating your PHP files, you can also
+define a number of directories that should be watched for file changes.
+This is useful for development environments.
+
+```caddyfile
+{
+    frankenphp {
+        worker /path/to/app/public/worker.php
```

Comment:
Currently it's global since all workers are restarted. I can change that, but it would be complicated. And how would the php_server command look then?

Reply:
The `php-server` is just a shortcut for convenience. It's not a problem to require to use a Caddyfile for advanced features. The goal isn't to support everything through this command.

Reply:
Watch is now a directive of worker 👍